rubocop/rubocop.yml

```diff
@@ -106,3 +108,14 @@ Metrics/ClassLength:
   Description: 'Avoid classes longer than 250 lines of code.'
   Enabled: false
   Max: 250
+
+Lint/RaiseException:
+    Enabled: false
+Lint/StructNewOverride:
+    Enabled: false
```

Comment:
I'm inclined to leave these first two enabled.
Pretty bad practice to break either of those.

Reply:
Yeah, good point. I've just enabled them.